aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/todo-group.csproj

```diff
@@ -6,11 +6,19 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <RootNamespace>todo_group</RootNamespace>
   </PropertyGroup>
-
+  
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.0-preview.5.22303.8" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="6.0.7" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.0-preview.7.22376.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.0-preview.7.22376.2">
```

Comment:
Can you update and test with V 7 RC1?

Reply:
sure, will do

Reply:
packages are updated to RC1. Please check `WebMinRouteGroup.csproj` as the project namespace has been updated